testing/spectest/shared/common/forkchoice/builder.go

```diff
@@ -39,26 +37,18 @@ func NewBuilder(t testing.TB, initialState state.BeaconState, initialBlock inter
 // Tick resets the genesis time to now()-tick and adjusts the slot to the appropriate value.
 func (bb *Builder) Tick(t testing.TB, tick int64) {
 	bb.service.SetGenesisTime(time.Unix(time.Now().Unix()-tick, 0))
+	bb.service.ForkChoicer().SetGenesisTime(uint64(time.Now().Unix() - tick))
```

Comment:
We need to keep track of genesis time in forkchoice now as well to update justification points because of the balancing attack. For proposer boost tests this becomes more important. I do not know how balancing attack tests were passing.